examples/exoskeleton/js/views/app-view.js

```diff
@@ -32,10 +41,10 @@ var app = app || {};
 		// collection, when items are added or changed. Kick things off by
 		// loading any preexisting todos that might be saved in *localStorage*.
 		initialize: function () {
-			this.allCheckbox = this.find('#toggle-all');
-			this.input = this.find('#new-todo');
-			this.footer = this.find('#footer');
-			this.main = this.find('#main');
+			this.allCheckbox = this.$('#toggle-all')[0];
```

Comment:
That looks like a regression, API-wise? Is there no longer a way to target a single node?